### ISSUE TITLE
error when initializing or confuiguring DBX : failure due to protobuf error: TypeError: Descriptors cannot not be created directly

### ISSUE THREAD

**Issue by AditiTwinkle** [databrickslabs/dbx#262]
## Expected Behavior
Should be able to initialize, configure DBX projects.

## Current Behavior
Throws an error of protobuf version downgrading
**dbx init**
```
Traceback (most recent call last):
  File "/mnt/c/Users/write/Documents/Aditi/dwh-databricks-acceleration-package/code_to_learn/venvlearncoding/bin/dbx", line 5, in <module>
    from dbx.cli import cli
  File "/mnt/c/Users/write/Documents/Aditi/dwh-databricks-acceleration-package/code_to_learn/venvlearncoding/lib/python3.8/site-packages/dbx/cli.py", line 4, in <module>
    from dbx.commands.configure import configure
  File "/mnt/c/Users/write/Documents/Aditi/dwh-databricks-acceleration-package/code_to_learn/venvlearncoding/lib/python3.8/site-packages/dbx/commands/configure.py", line 7, in <module>
    from dbx.utils.common import (
  File "/mnt/c/Users/write/Documents/Aditi/dwh-databricks-acceleration-package/code_to_learn/venvlearncoding/lib/python3.8/site-packages/dbx/utils/common.py", line 14, in <module>
    import mlflow
  File "/mnt/c/Users/write/Documents/Aditi/dwh-databricks-acceleration-package/code_to_learn/venvlearncoding/lib/python3.8/site-packages/mlflow/__init__.py", line 32, in <module>
    import mlflow.tracking._model_registry.fluent
  File "/mnt/c/Users/write/Documents/Aditi/dwh-databricks-acceleration-package/code_to_learn/venvlearncoding/lib/python3.8/site-packages/mlflow/tracking/__init__.py", line 8, in <module>
    from mlflow.tracking.client import MlflowClient
  File "/mnt/c/Users/write/Documents/Aditi/dwh-databricks-acceleration-package/code_to_learn/venvlearncoding/lib/python3.8/site-packages/mlflow/tracking/client.py", line 16, in <module>
    from mlflow.entities import Experiment, Run, RunInfo, Param, Metric, RunTag, FileInfo, ViewType
  File "/mnt/c/Users/write/Documents/Aditi/dwh-databricks-acceleration-package/code_to_learn/venvlearncoding/lib/python3.8/site-packages/mlflow/entities/__init__.py", line 6, in <module>
    from mlflow.entities.experiment import Experiment
  File "/mnt/c/Users/write/Documents/Aditi/dwh-databricks-acceleration-package/code_to_learn/venvlearncoding/lib/python3.8/site-packages/mlflow/entities/experiment.py", line 2, in <module>
    from mlflow.entities.experiment_tag import ExperimentTag
  File "/mnt/c/Users/write/Documents/Aditi/dwh-databricks-acceleration-package/code_to_learn/venvlearncoding/lib/python3.8/site-packages/mlflow/entities/experiment_tag.py", line 2, in <module>
    from mlflow.protos.service_pb2 import ExperimentTag as ProtoExperimentTag
  File "/mnt/c/Users/write/Documents/Aditi/dwh-databricks-acceleration-package/code_to_learn/venvlearncoding/lib/python3.8/site-packages/mlflow/protos/service_pb2.py", line 18, in <module>
    from .scalapb import scalapb_pb2 as scalapb_dot_scalapb__pb2
  File "/mnt/c/Users/write/Documents/Aditi/dwh-databricks-acceleration-package/code_to_learn/venvlearncoding/lib/python3.8/site-packages/mlflow/protos/scalapb/scalapb_pb2.py", line 29, in <module>
    options = _descriptor.FieldDescriptor(
  File "/mnt/c/Users/write/Documents/Aditi/dwh-databricks-acceleration-package/code_to_learn/venvlearncoding/lib/python3.8/site-packages/google/protobuf/descriptor.py", line 560, in __new__
    _message.Message._CheckCalledFromGeneratedFile()
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).

```

## Steps to Reproduce (for bugs)
install the latest version on ubuntu via WSL

## Context
Setting up a dbx project from scratch

## Your Environment
N/A

* dbx version used:
* Databricks Runtime version:

**Comment by matthayes** [comment]
This looks like a duplicate of #257.  If you run `pip list` you may find you have picked up a RC of protobof `4.21.0rc2` that appears to be incompatible.  This was addressed in e6361bf922813b94ae47bc6aee308a05c80d6da4.  As a workaround you could try fixing the protobuf version at `protobuf==3.20.1`.

**Comment by AditiTwinkle** [comment]
works with downgrading to 3.20.1
